script/entrypoint.sh

```diff
@@ -46,7 +46,7 @@ mv -v /etc/nginx/conf.d /etc/nginx/conf.d.disabled
     mv -v /etc/nginx/conf.d.disabled /etc/nginx/conf.d #enable
     echo "reload nginx with ssl"
     nginx -s reload
-    sleep 60d
+    sleep 1d
```

Comment:
this will reload nginx every day. not sure how smart nginx handles reloads but I think it will be safer to add status code of le.run and skip reload if != 0

Reply:
From my experience this should not be a problem at all. The documentation also says:

>Configuration reload
>Start the new worker processes with a new configuration
>**Gracefully** shutdown the old worker processes

So this shouldn't kill any active connections.

Reply:
What will happen if service.conf changed to something broken from syntax point of view? E.g. I'm editing file right now and nginx think it's time to reload?
With old behavior it happens once in two months, now it could happen every day.

Reply:
nginx will keep running the old config file if reload is triggered while the current config is invalid.

Also, why would you be editing the file on production, anyway? :)

Reply:
I tend to agree with @paskal - unneeded reloads are better to be avoided. We don't know how users manage they servers. If someone is brave enough to do it directly - fine with me, do whatever you want with your own server :) I think we should just eliminate daily reloads and perform it only in case of certs update/generation.